### PR TITLE
Remove AMP-Access-Control-Allow-Source-Origin 

### LINKED ISSUE #382: [toolbox-cors] Remove AMP specific header handling
The only features that are still needed are:

* standard CORS headers
* AMP-redirect-to

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes #382

### CODE REVIEW

File: packages/cors/README.md [hunk 1]
```diff
@@ -104,21 +103,16 @@ $ curl --header "AMP-SAME-ORIGIN: true" -I "http://localhost:3000/items?__amp_so
...
-AMP-Access-Control-Allow-Source-Origin: https://localhost:3000
 Content-Type: application/json; charset=utf-8
 ...
 ```
 
 2. AMP CORS header will be set if the `__amp_source_origin` query parameter is set together with the `Origin` header:
 
 ```
-$ curl --header "Origin: https://ampbyexample-com.cdn.ampproject.org" -I "http://localhost:3000/items?__amp_source_origin=https://localhost:3000"
+$ curl --header "Origin: https://amp.dev.cdn.ampproject.org" -I "http://localhost:3000/items?__amp_source_origin=https://localhost:3000"
```
Comment (ithinkihaveacat) [inline packages/cors/README.md]: Should be `amp-dev.cdn.ampproject.org`?

File: packages/cors/README.md [hunk 2]
```diff
@@ -104,21 +103,16 @@ $ curl --header "AMP-SAME-ORIGIN: true" -I "http://localhost:3000/items?__amp_so
...
 HTTP/1.1 200 OK
-X-Powered-By: Express
-Access-Control-Allow-Origin: https://ampbyexample-com.cdn.ampproject.org
-Access-Control-Expose-Headers: AMP-Access-Control-Allow-Source-Origin
-AMP-Access-Control-Allow-Source-Origin: https://localhost:3000
+Access-Control-Allow-Origin: https://amp.dev.cdn.ampproject.org
```
Comment (ithinkihaveacat) [inline packages/cors/README.md]: `amp-dev.cdn.ampproject.org`?

File: packages/cors/lib/cors.js [hunk 3]
```diff
@@ -78,12 +78,13 @@ module.exports = (options, caches=new Caches()) => {
...
-    const headersToExpose = ['AMP-Access-Control-Allow-Source-Origin'];
+    const headersToExpose = [];
     if (options.enableAmpRedirectTo) {
       headersToExpose.push('AMP-Redirect-To');
     }
-    response.setHeader('Access-Control-Expose-Headers', headersToExpose);
-    response.setHeader('AMP-Access-Control-Allow-Source-Origin', sourceOrigin);
+    if (headersToExpose.length > 0) {
+      response.setHeader('Access-Control-Expose-Headers', headersToExpose);
+    }
```
Comment (ithinkihaveacat) [inline packages/cors/lib/cors.js]: I think `headersToExpose` can be removed, this is now equivalent to:

```js
if (options.enableAmpRedirectTo) {
  response.setHeader('Access-Control-Expose-Headers', 'AMP-Redirect-To');
}
```

File: packages/cors/README.md (hunk 1 above)
Comment (sebastianbenz) [reply packages/cors/README.md]: yes! done

File: packages/cors/README.md (hunk 2 above)
Comment (sebastianbenz) [reply packages/cors/README.md]: done

File: packages/cors/lib/cors.js (hunk 3 above)
Comment (sebastianbenz) [reply packages/cors/lib/cors.js]: done

**Pull request merged**